tracer/src/Datadog.Trace/FrameworkDescription.NetCore.cs

```diff
@@ -93,12 +93,19 @@ private static string GetNetCoreOrNetFrameworkVersion()
             {
                 try
                 {
+#if NETFX
```

Comment:
What is `NETFX`?

Reply:
This entire file is .NET Core only, so I think you can just change `RootAssembly.CodeBase` to `RootAssembly.Location` below, if that was the intent.

Reply:
Fixed